doc/extension.rst

```diff
@@ -121,6 +81,14 @@ The extension has a few configuration options that can be set in ``conf.py``:
    You can also pass in the compiler to use, for example
    ``get_include_args('gcc')``.
 
+.. py:data:: cautodoc_transformations
```

Comment:
So this is the 2nd definition of this symbol in this commit (other one in `built-in-extensions`) and Sphinx dutifully complains about it. Haven't read through this carefully yet, but this looks like some sort of mistake.

Reply:
The `:noindex:` here should avoid the complaints, and prevent this from being a link target. Sometimes Sphinx caching causes an issue when things change; try `make clean`.

Reply:
Indeed, `noindex` does solve the compilation issue (caching aside), but it's a hack. This is now a configuration variable of the new extension not of hawkmoth itself, which is the scope of this section.

Maybe we could replace this with a note informing that hawkmoth currently enables a built in extension by default to assist in transition and link it there so the user can find additional configuration options. It would be more accurate and avoid this oddity.

Note that the compiled docs (with links) are ok, but if you took the plain text or a printed format, the link below would still be a bit ambiguous.

Reply:
> Maybe we could replace this with a note informing that hawkmoth currently enables a built in extension by default to assist in transition and link it there so the user can find additional configuration options.

Heh, well, the above was the goal of adding this here.

Reply:
What do you think of, instead of:
```rst
.. py:data:: variable_that_does_not_really_belong_here_anymore
   :hack:
```
this:
```rst
.. note::

   For backwards compatibility, hawkmoth automatically includes the extention
   :extension:`something`, which itself exposes additional configuration options.
```

Possibly adding 'This automatic inclusion and the extension itself is planned to be removed later on.' depending on how you want to handle that.

Reply:
Resolved in #131 by just adding a reference to configuration in built-in extensions.